_sass/index.scss

```diff
@@ -1,17 +1,36 @@
-.hero-section {
-  font-family: 'Helvetica Neue';
+body {
+  margin: 0;
+}
 
-  &__headline {
-    font-size: 1.75rem;
-    margin-bottom: -7rem;
-    width: 21rem;
-    position: relative;
-    z-index: 2;
+.header {
+  color: #ffffff;
+  font-family: 'Swift Neue LT Pro';
```

Comment:
Consider switching over to the `$font_family-primary` variable for consistency, likewise switching line 8's hex code to the `$color_gold` variable.